extra/release_tool.py

```diff
@@ -193,8 +191,8 @@ def is_release_component(self):
 # categorize backend services wrt open/enterprise versions
 # important for test suite selection
 BACKEND_SERVICES_OPEN = {"deviceauth", "inventory"}
-BACKEND_SERVICES_ENT = {"deployments-enterprise", "mender-conductor-enterprise", "tenantadm", "useradm-enterprise"}
-BACKEND_SERVICES_OPEN_ENT = {"deployments", "mender-conductor", "useradm"}
+BACKEND_SERVICES_ENT = {"deployments-enterprise", "tenantadm", "useradm-enterprise"}
+BACKEND_SERVICES_OPEN_ENT = {"deployments", "useradm"}
```

Comment:
Unrelated to this PR, sorry, but aren't wee missing here `inventory-enterprise` update?

I think it should be something like:
```
BACKEND_SERVICES_OPEN = {"deviceauth"}
BACKEND_SERVICES_ENT = {"deployments-enterprise", "tenantadm", "useradm-enterprise", "inventory-enterprise"}
BACKEND_SERVICES_OPEN_ENT = {"deployments", "useradm", "inventory"}
```

Reply:
Done.